.github/workflows/test.yml

```diff
@@ -42,7 +57,17 @@ jobs:
         python -m pip install --upgrade pip wheel
         python -m pip install --upgrade tox
 
+    - name: Set vendor environment
+      if: ${{ matrix.vendored_env != 'unvendored' }}
+      shell: bash
+      run: |
+        echo "DANDI_INSTANCE_NAME=${{ matrix.instance_name }}" >> "$GITHUB_ENV"
+        echo "DANDI_DATACITE_DOI_ID=${{ matrix.datacite_doi_id }}" >> "$GITHUB_ENV"
+
     - name: Run all tests
+      env:
+        DATACITE_DEV_PASSWORD: ${{ secrets.DATACITE_DEV_PASSWORD }}
+        NO_ET: 1
```

Comment:
if ember folks would provide password, we would need to make this specific to each vendored instance since DANDI and EMBER passwords would be different.

Also add variable `DATACITE_DEV_LOGIN` which is currently hardcoded to be `DARTLIB.DANDI` (DANDI specific), just for testing

Reply:
I am sending your our datacite information in an email @yarikoptic 
use **dataciteinfo** to unencrypt it.